questions/app-transparent-background.question.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "How can I set a transparent app background?"
+alt_titles:
+  - "Transparent background in the app"
+  - "See-through background"
+  - "See-through app"
+  - "Translucid app background"
+  - "Textual apps are opaque but terminal is transparent"
+---
+
+Textual does not provide a mechanism to make your applications have a transparent background.
```

Comment:
The working makes it sound like we are intentionally disabling this feature, when it is more a side effect of a requirement to make apps look good everywhere.

Reply:
I reworded the answer to try and align it with your suggestion.